secboot/secboot.go

```diff
@@ -186,6 +186,11 @@ type ResealKeysParams struct {
 	TPMPolicyAuthKeyFile string
 }
 
+type ResealKeysParamsForHooks struct {
```

Comment:
does this need a doc comment?  can you explain why it is not used here but only defined?